### ISSUE TITLE
Add GitHub Actions for CI

### ISSUE THREAD

**Issue by szabgab** [jimav/ODF-lpOD_Helper#2]
I am using the CPAN Digger to locate recently release modules that don't have a CI system. As I can see [non of your modules](https://cpan-digger.perlmaven.com/author/JIMAVERA) have a CI setup.

This project too could benefit from setting up a CI system to run the tests on every push and every pull-requests. I've written about this [here](https://perlmaven.com/github-actions).

If you are interested I'd be happy to send you PR to this and to your other CPAN modules.

**Comment by jimav** [comment]
Hi, Could you summarize what this "CI" setup would provide beyond the current system where CPAN smokers email test results?

**Comment by szabgab** [comment]
The main thing is that it can run the tests on every push, long before the release.

It can also have a lot more "involved" environment (e.g. a database set up) something the CPAN testers generally don't have. I am not sure if this is relevant to any of you modules, though.
